pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/PubNub.kt

```diff
@@ -463,44 +473,122 @@ interface PubNub : EventEmitter, StatusEmitter {
     /**
      * The method returns a list of channel memberships for a user. This method doesn't return a user's subscriptions.
      */
+    @Deprecated(
+        message = "This function is deprecated. Use getMemberships(userId: String) for better functionality.",
+        replaceWith = ReplaceWith(
+            "getMemberships(userId)"
+        )
+    )
     fun getMemberships(): GetMemberships
 
+    /**
+     * The method returns a list of channel memberships for a user. This method doesn't return a user's subscriptions.
+     */
+    fun getMemberships(userId: String): GetMembershipsBuilder
```

Comment:
Good point. I will remove GetMembershipsBuild and reuse GetMemberships because there can't be two methods 
fun getMemberships() with different return types.